src/apps/backend/modules/account/account_service.py

```diff
@@ -80,3 +80,7 @@ def create_or_update_account_notification_preferences(
     @staticmethod
     def get_account_notification_preferences_by_account_id(*, account_id: str) -> AccountNotificationPreferences:
         return NotificationService.get_account_notification_preferences_by_account_id(account_id=account_id)
+
+    @staticmethod
+    def delete_account(*, account_id: str) -> Account:
```

Comment:
Since account is deleted, it is not expected for this function return `Account` object